### ISSUE TITLE
Crash with 1.23.0-dev.beta.1

### ISSUE THREAD

**Issue by casella** [OpenModelica/OpenModelica#12503]
From a received crash report:
- load Buildings lib 11.0.0
- navigate to `Buildings.ThermalZones.Detailed.Examples.MixedAirCO2`
- right click on `roo` component and open class

@adeas31 could you please check that? It could be worth fixing before the release. If it's complicated, we keep it for 1.23.1.

**Comment by adeas31** [comment]
This works fine for me.

![image](https://github.com/OpenModelica/OpenModelica/assets/4007231/fac2dc2a-ee3b-49ce-ac44-b272af1722b2)

No idea why it crashed. The report shows that it crashed inside the Qt sources.

**Comment by casella** [comment]
OK, I wrote to the crash reporter. In case, we can reopen it.

**Comment by n-marci** [comment]
Hello I submitted the original report.
I can reproduce the crash with the same steps as above on a fresh Ubuntu 24.04 install (everything up to date) inside a [distrobox](https://github.com/89luca89/distrobox) container.
My desktop is gnome wayland.

Not sure if it helps but these are the system logs:
```
Jun 05 14:12:54 desktop gnome-shell[2202]: Window manager warning: Ping serial 21725410 was reused for window W155, previous use was for window 0x160000f.
Jun 05 14:13:07 desktop gnome-shell[2202]: Window manager warning: Invalid WM_TRANSIENT_FOR window 0x1600008 specified for 0x1600024.
Jun 05 14:13:28 desktop gnome-shell[2202]: Window manager warning: Invalid WM_TRANSIENT_FOR window 0x1600008 specified for 0x160002a.
Jun 05 14:13:31 desktop gnome-shell[2202]: Window manager warning: Invalid WM_TRANSIENT_FOR window 0x1600008 specified for 0x1600030.
Jun 05 14:13:48 desktop systemd[1]: Started Process Core Dump (PID 32287/UID 0).
Jun 05 14:13:48 desktop systemd-timesyncd[802]: Network configuration changed, trying to establish connection.
Jun 05 14:13:48 desktop systemd-timesyncd[802]: Network configuration changed, trying to establish connection.
Jun 05 14:13:48 desktop systemd-timesyncd[802]: Network configuration changed, trying to establish connection.
Jun 05 14:13:48 desktop systemd-timesyncd[802]: Contacted time server [2003:a:87f:c37c::7]:123 (2.nixos.pool.ntp.org).
Jun 05 14:13:59 desktop systemd-coredump[32288]: [🡕] Process 31887 (OMEdit) of user 1000 dumped core.
                                                 
                                                 Stack trace of thread 31887:
                                                 #0  0x00007f67bcd621a4 n/a (n/a + 0x0)
                                                 ELF object binary architecture: AMD x86-64
Jun 05 14:13:59 desktop systemd[1]: systemd-coredump@2-32287-0.service: Deactivated successfully.
Jun 05 14:13:59 desktop systemd[1]: systemd-coredump@2-32287-0.service: Consumed 2.983s CPU time, no IP traffic.

```

**Comment by casella** [comment]
@adeas31 can you please check?

**Comment by adeas31** [comment]
I will try it on WSL.

**Comment by adeas31** [comment]
@n-marci did you compile OM from source or install it using apt-get?

**Comment by n-marci** [comment]
I installed with apt-get.

**Comment by perost** [comment]
I also couldn't reproduce the issue, in my case on Arch Linux. However, from the stack trace it looks like it's crashing when printing out errors after calling `getModelInstance`, and I get a lot of JSON parsing errors when opening that class. Maybe it tries to print out more messages than Qt in that particular environment can handle?

But anyway, I opened #12514 for the issue that the output of `getModelInstance` contains invalid expressions. Fixing that issue may stop OMEdit from crashing in this particular case by eliminating the error messages, but the general issue would still remain in that case.

**Comment by adeas31** [comment]
This works fine on WSL as well,

```
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.4 LTS
Release:        22.04
Codename:       jammy
```

```
QMake version 3.1
Using Qt version 5.15.3 in /usr/lib/x86_64-linux-gnu
```

![image](https://github.com/OpenModelica/OpenModelica/assets/4007231/53d8c218-1dbf-4e36-a920-35ed6584050a)

Maybe we can try again after #12514 is fixed.

**Comment by adeas31** [comment]
The release version `OpenModelica 1.23.0~dev.beta.1-1-g379f714` from apt is crashing. I updated to nightly `OpenModelica 1.24.0~dev-103-g1ef9b1b` and it worked fine.

@adrpo any idea? Maybe we forgot to port some commit or something else is missing with the release version.

**Comment by adrpo** [comment]
> The release version `OpenModelica 1.23.0~dev.beta.1-1-g379f714` from apt is crashing. I updated to nightly `OpenModelica 1.24.0~dev-103-g1ef9b1b` and it worked fine.
> 
> @adrpo any idea? Maybe we forgot to port some commit or something else is missing with the release version.

No idea. Can you point me to the crash report?

**Comment by adeas31** [comment]
https://dev.openmodelica.org/omeditcrashreports/tmp/2024-06-10_15:25:17-openmodelica.stacktrace.OMEdit

**Comment by adrpo** [comment]
As far as I can tell it runs out of memory.

**Comment by adrpo** [comment]
If I got directly to Buildings.ThermalZones.Detailed.MixedAir (which is the roo component) I get this. Maybe this can point into the right direction.

```json
[4] 12:12:49 Scripting Notification
Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[{"$kind":"sub","exp":[{"$kind":"binary_op","lhs":0.134831,"op":"/","rhs":0}],"subscripts":["<lay, 1>"]}],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        {
            "$kind": "sub",
            "exp": [
                {
                    "$kind": "binary_op",
                    "lhs": 0.134831,
                    "op": "/",
                    "rhs": 0
                }
            ],
            "subscripts": [
                "<lay, 1>"
            ]
        }
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[2],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        2
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[3],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        3
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[[0,0]],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        [
            0,
            0
        ]
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[{"$kind":"sub","exp":[[{"$kind":"binary_op","lhs":1,"op":"/","rhs":0},{"$kind":"binary_op","lhs":1,"op":"/","rhs":0}]],"subscripts":["<lay, 1>"]}],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        {
            "$kind": "sub",
            "exp": [
                [
                    {
                        "$kind": "binary_op",
                        "lhs": 1,
                        "op": "/",
                        "rhs": 0
                    },
                    {
                        "$kind": "binary_op",
                        "lhs": 1,
                        "op": "/",
                        "rhs": 0
                    }
                ]
            ],
            "subscripts": [
                "<lay, 1>"
            ]
        }
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[[0,0]],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        [
            0,
            0
        ]
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[{"$kind":"sub","exp":[[{"$kind":"binary_op","lhs":1,"op":"/","rhs":0},{"$kind":"binary_op","lhs":1,"op":"/","rhs":0}]],"subscripts":["<lay, 1>"]}],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        {
            "$kind": "sub",
            "exp": [
                [
                    {
                        "$kind": "binary_op",
                        "lhs": 1,
                        "op": "/",
                        "rhs": 0
                    },
                    {
                        "$kind": "binary_op",
                        "lhs": 1,
                        "op": "/",
                        "rhs": 0
                    }
                ]
            ],
            "subscripts": [
                "<lay, 1>"
            ]
        }
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[{"$kind":"sub","exp":[{"$kind":"binary_op","lhs":0.134831,"op":"/","rhs":0}],"subscripts":["<lay, 1>"]}],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        {
            "$kind": "sub",
            "exp": [
                {
                    "$kind": "binary_op",
                    "lhs": 0.134831,
                    "op": "/",
                    "rhs": 0
                }
            ],
            "subscripts": [
                "<lay, 1>"
            ]
        }
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[2],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        2
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[3],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        3
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[[0,0]],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        [
            0,
            0
        ]
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[{"$kind":"sub","exp":[[{"$kind":"binary_op","lhs":1,"op":"/","rhs":0},{"$kind":"binary_op","lhs":1,"op":"/","rhs":0}]],"subscripts":["<lay, 1>"]}],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        {
            "$kind": "sub",
            "exp": [
                [
                    {
                        "$kind": "binary_op",
                        "lhs": 1,
                        "op": "/",
                        "rhs": 0
                    },
                    {
                        "$kind": "binary_op",
                        "lhs": 1,
                        "op": "/",
                        "rhs": 0
                    }
                ]
            ],
            "subscripts": [
                "<lay, 1>"
            ]
        }
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[[0,0]],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        [
            0,
            0
        ]
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[{"$kind":"sub","exp":[[{"$kind":"binary_op","lhs":1,"op":"/","rhs":0},{"$kind":"binary_op","lhs":1,"op":"/","rhs":0}]],"subscripts":["<lay, 1>"]}],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        {
            "$kind": "sub",
            "exp": [
                [
                    {
                        "$kind": "binary_op",
                        "lhs": 1,
                        "op": "/",
                        "rhs": 0
                    },
                    {
                        "$kind": "binary_op",
                        "lhs": 1,
                        "op": "/",
                        "rhs": 0
                    }
                ]
            ],
            "subscripts": [
                "<lay, 1>"
            ]
        }
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[0.16],"subscripts":["<glass, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        0.16
    ],
    "subscripts": [
        "<glass, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[0.16],"subscripts":["<glass, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        0.16
    ],
    "subscripts": [
        "<glass, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[{"$kind":"binary_op","lhs":0.0015,"op":"/","rhs":0}],"subscripts":["<glass, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        {
            "$kind": "binary_op",
            "lhs": 0.0015,
            "op": "/",
            "rhs": 0
        }
    ],
    "subscripts": [
        "<glass, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[{"$kind":"sub","exp":[{"$kind":"binary_op","lhs":0.134831,"op":"/","rhs":0}],"subscripts":["<lay, 1>"]}],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        {
            "$kind": "sub",
            "exp": [
                {
                    "$kind": "binary_op",
                    "lhs": 0.134831,
                    "op": "/",
                    "rhs": 0
                }
            ],
            "subscripts": [
                "<lay, 1>"
            ]
        }
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[2],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        2
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[3],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        3
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[[0,0]],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        [
            0,
            0
        ]
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[{"$kind":"sub","exp":[[{"$kind":"binary_op","lhs":1,"op":"/","rhs":0},{"$kind":"binary_op","lhs":1,"op":"/","rhs":0}]],"subscripts":["<lay, 1>"]}],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        {
            "$kind": "sub",
            "exp": [
                [
                    {
                        "$kind": "binary_op",
                        "lhs": 1,
                        "op": "/",
                        "rhs": 0
                    },
                    {
                        "$kind": "binary_op",
                        "lhs": 1,
                        "op": "/",
                        "rhs": 0
                    }
                ]
            ],
            "subscripts": [
                "<lay, 1>"
            ]
        }
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[[0,0]],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        [
            0,
            0
        ]
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[{"$kind":"sub","exp":[[{"$kind":"binary_op","lhs":1,"op":"/","rhs":0},{"$kind":"binary_op","lhs":1,"op":"/","rhs":0}]],"subscripts":["<lay, 1>"]}],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        {
            "$kind": "sub",
            "exp": [
                [
                    {
                        "$kind": "binary_op",
                        "lhs": 1,
                        "op": "/",
                        "rhs": 0
                    },
                    {
                        "$kind": "binary_op",
                        "lhs": 1,
                        "op": "/",
                        "rhs": 0
                    }
                ]
            ],
            "subscripts": [
                "<lay, 1>"
            ]
        }
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[{"$kind":"sub","exp":[{"$kind":"binary_op","lhs":0.134831,"op":"/","rhs":0}],"subscripts":["<lay, 1>"]}],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        {
            "$kind": "sub",
            "exp": [
                {
                    "$kind": "binary_op",
                    "lhs": 0.134831,
                    "op": "/",
                    "rhs": 0
                }
            ],
            "subscripts": [
                "<lay, 1>"
            ]
        }
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[2],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        2
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[3],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        3
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[[0,0]],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        [
            0,
            0
        ]
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[{"$kind":"sub","exp":[[{"$kind":"binary_op","lhs":1,"op":"/","rhs":0},{"$kind":"binary_op","lhs":1,"op":"/","rhs":0}]],"subscripts":["<lay, 1>"]}],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        {
            "$kind": "sub",
            "exp": [
                [
                    {
                        "$kind": "binary_op",
                        "lhs": 1,
                        "op": "/",
                        "rhs": 0
                    },
                    {
                        "$kind": "binary_op",
                        "lhs": 1,
                        "op": "/",
                        "rhs": 0
                    }
                ]
            ],
            "subscripts": [
                "<lay, 1>"
            ]
        }
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[[0,0]],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        [
            0,
            0
        ]
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} Failed to deserialize json:  QJsonValue(object, QJsonObject({"$kind":"sub","exp":[{"$kind":"sub","exp":[[{"$kind":"binary_op","lhs":1,"op":"/","rhs":0},{"$kind":"binary_op","lhs":1,"op":"/","rhs":0}]],"subscripts":["<lay, 1>"]}],"subscripts":["<lay, 1>"]}))
Expression: invalid JSON object {
    "$kind": "sub",
    "exp": [
        {
            "$kind": "sub",
            "exp": [
                [
                    {
                        "$kind": "binary_op",
                        "lhs": 1,
                        "op": "/",
                        "rhs": 0
                    },
                    {
                        "$kind": "binary_op",
                        "lhs": 1,
                        "op": "/",
                        "rhs": 0
                    }
                ]
            ],
            "subscripts": [
                "<lay, 1>"
            ]
        }
    ],
    "subscripts": [
        "<lay, 1>"
    ]
} 


```

**Comment by adrpo** [comment]
Strangely, if I open the class first then do the thing in the ticket it doesn't crash anymore.

**Comment by adeas31** [comment]
See https://github.com/OpenModelica/OpenModelica/issues/12514. Maybe fixing it can fix this issue.

But something has definitely changed since it doesn't run out of memory on master. Perhaps @perost missed porting a commit to 1.23

**Comment by adrpo** [comment]
That could be. I need to build a debug version of it to understand why it dies as the stacktrace points to it but it doesn't help with actual values.

```
#0  0x00007ffff199bb08 in QTextEngine::shapeTextWithHarfbuzzNG(QScriptItem const&, unsigned short const*, int, QFontEngine*, QVector<unsigned int> const&, bool, bool) const () at /lib/x86_64-linux-gnu/libQt5Gui.so.5
#1  0x00007ffff199e637 in QTextEngine::shapeText(int) const () at /lib/x86_64-linux-gnu/libQt5Gui.so.5
#2  0x00007ffff19b06e1 in QTextLine::layout_helper(int) () at /lib/x86_64-linux-gnu/libQt5Gui.so.5
#3  0x00007ffff19f88d5 in  () at /lib/x86_64-linux-gnu/libQt5Gui.so.5
#4  0x00007ffff19f9cc5 in  () at /lib/x86_64-linux-gnu/libQt5Gui.so.5
#5  0x00007ffff1a01ae7 in  () at /lib/x86_64-linux-gnu/libQt5Gui.so.5
#6  0x00007ffff1a020de in  () at /lib/x86_64-linux-gnu/libQt5Gui.so.5
#7  0x00007ffff1a02708 in QTextDocumentLayout::doLayout(int, int, int) () at /lib/x86_64-linux-gnu/libQt5Gui.so.5
#8  0x00007ffff1a03a9b in QTextDocumentLayout::documentChanged(int, int, int) () at /lib/x86_64-linux-gnu/libQt5Gui.so.5
#9  0x00007ffff19d8d73 in QTextDocumentPrivate::finishEdit() () at /lib/x86_64-linux-gnu/libQt5Gui.so.5
#10 0x00007ffff1a0fb53 in QTextCursor::insertFragment(QTextDocumentFragment const&) () at /lib/x86_64-linux-gnu/libQt5Gui.so.5
#11 0x00007ffff1a0fdf0 in QTextCursor::insertHtml(QString const&) () at /lib/x86_64-linux-gnu/libQt5Gui.so.5
#12 0x00005555557f1a7a in MessageWidget::addGUIMessage(MessageItem) ()
#13 0x00005555557f2f73 in MessagesWidget::addGUIMessage(MessageItem) ()
#14 0x00005555557bfd0f in MainWindow::printStandardOutAndErrorFilesMessages() ()
#15 0x00005555557d5aed in OMCProxy::printMessagesStringInternal() ()
#16 0x00005555557ecb86 in OMCProxy::getModelInstance(QString const&, QString const&, bool, bool) ()
#17 0x000055555583a95d in ModelWidget::loadModelInstance(bool, ModelInfo const&) ()
#18 0x000055555583e947 in ModelWidget::loadDiagramViewNAPI() ()
#19 0x0000555555856320 in ModelWidgetContainer::addModelWidget(ModelWidget*, bool) ()
#20 0x00005555557f9c78 in LibraryTreeModel::showModelWidget(LibraryTreeItem*, bool) ()
#21 0x000055555590b4e5 in Element::openClass() ()

```

**Comment by adrpo** [comment]
Seems to be fixed with v1.23.1.
